Alesab/cosmos (#4)
CAPIBARA3/cosmos-papers: COSMOS-Programme/cosmos.tex

Reviewer: ```suggestion
	\left(\begin{matrix} \mathbf{B}_{21} \\ \mathbf{B}_{31} \end{matrix} \right) \hat{s} = c \left(\begin{matrix}\Delta t_{21} \\ \Delta t_{31}\end{matrix}\right)
```

The key innovation of the COSMOS programme is to leverage the already developed high-energy observing capabilities from heritage missions and introduce arcminute-level source localisation via coordinated observations. This is a known concept in physics, for instance various GW observatories (LIGO, VIRGO and KAGRA) together can further constraint the sky coordinates of a signal than a single one \cite{missing}. The interplanetary network (IPN) \cite{missing} also used this technique with high-energy space telescopes to locate astrophysical sources. Now, we want to leverage the existing CubeSat technology for high energy astronomy to provide such measurements.

> The main idea is that comparing the signals from the same source detected by different satellites we can tighter constrain the origin of the source, in terms of celestial coordinates. For two satellites separated at baseline distance $\mathbf{B}$ and the time-delay between both detectors is $\delta t = t_2 - t_1$ then the direction of the source in the sky is given by %SUGGESTION:use \Delta instead of \delta

Reviewer: ```suggestion
The main idea is that comparing the signals from the same source detected by different satellites we can tighter constrain the origin of the source, in terms of celestial coordinates. For two satellites separated at baseline distance $\mathbf{B}$ and the time-delay between both detectors is $\Delta t = t_2 - t_1$ then the direction of the source in the sky is given by 
```

Reviewer: Sure!

	\sigma_\theta = \frac{1}{|\mathbf{B}_\perp|} \sqrt{(c \sigma_t)^2 + \sigma_{B, ||} ^2}.
\end{equation}

For reference, given a time uncertainty $\sigma_t = 100\,\mathrm{ns}$ and a parallel part of B uncertainty $\sigma_{B, \parallel} \approx 10\,\mathrm{m}$, we obtain:

\begin{align}
\sigma_\theta
&= \frac{1}{|\mathbf{B}_\perp|} \sqrt{\left( 3 \times 10^{8}\,\mathrm{m/s} \times 10^{-7}\,\mathrm{s} \right)^2 + (10\,\mathrm{m})^2} \\[4pt]
&= \frac{\sqrt{(30\,\mathrm{m})^2 + (10\,\mathrm{m})^2}}{|\mathbf{B}_\perp|} \\[4pt]
&= \frac{31.6\,\mathrm{m}}{|\mathbf{B}_\perp|}
\end{align}

> We deduce that $\sigma_\theta$ is inversely proportional to $|\mathbf{B}_\perp|$; as the baseline gets larger, the uncertainty decreases. To estimate it, we assign $|\mathbf{B}_\perp|$ some numerical values: $|\mathbf{B}_\perp| = 100\,\mathrm{m}, 250\,\mathrm{m}, 500\,\mathrm{m}, 1000\,\mathrm{m}$.

Reviewer: ```suggestion
Note that that $\sigma_\theta \sim \frac{1}{\mathbf{B}_\perp$; as the baseline gets larger, the uncertainty decreases.
```

\begin{align}
\sigma_{\theta,1} &= \frac{31.6\,\mathrm{m}}{100\,\mathrm{m}} = 0.316\,\mathrm{rad} \approx 18.1^\circ \\
\sigma_{\theta,2} &= \frac{31.6\,\mathrm{m}}{250\,\mathrm{m}} = 0.126\,\mathrm{rad} \approx 7.22^\circ \\
\sigma_{\theta,3} &= \frac{31.6\,\mathrm{m}}{500\,\mathrm{m}} = 6.32 \times 10^{-2}\,\mathrm{rad} \approx 3.62^\circ \\
\sigma_{\theta,4} &= \frac{31.6\,\mathrm{m}}{1000\,\mathrm{m}} = 3.16 \times 10^{-2}\,\mathrm{rad} \approx 1.81^\circ
\end{align}

> 

Reviewer: ```suggestion

```

Author: Sorry Joan, I just want to know if this has to go because I got it wrong and it wasn't what I was supposed to do or because I have to do it in another way

Reviewer: Hi @alesab-08, you did nothing wrong, but instead of this set of formulas I think it is better to put your results in a table, which I put in another content. Thus the content doesn’t has to go, only the format.

	\sigma_\theta = \frac{1}{|\mathbf{B}_\perp|} \sqrt{(c \sigma_t)^2 + \sigma_{B, ||} ^2}.
\end{equation}

For reference, given $\sigma_t = 100\\mathrm{ns}$ and $\sigma_{B, \parallel} \approx 10\\mathrm{m}$, we obtain:

\begin{align}
\sigma_\theta
&= \frac{1}{|\mathbf{B}_\perp|} \sqrt{\left( 3 \times 10^{8}\,\mathrm{m/s} \times 10^{-7}\,\mathrm{s} \right)^2 + (10\,\mathrm{m})^2} \\[4pt]
&= \frac{\sqrt{(30\,\mathrm{m})^2 + (10\,\mathrm{m})^2}}{|\mathbf{B}_\perp|} \\[4pt]
&= \frac{31.6\,\mathrm{m}}{|\mathbf{B}_\perp|}
\end{align}

\begin{table}[h]
    \centering
    \begin{tabular}{cccc}
        \hline
        \textbf{Case} & \textbf{Distance (m)} & $\boldsymbol{\sigma_\theta}$ \textbf{(rad)} & $\boldsymbol{\sigma_\theta}$ \textbf{(degrees)} \\
        \hline
        1 & 100  & 0.316  & 18.1 \\
        2 & 250  & 0.126  & 7.22 \\
        3 & 500  & 0.0632 & 3.62 \\
        4 & 1000 & 0.0316 & 1.81 \\
        \hline

>     \end{tabular}

Reviewer: ```suggestion
    \end{tabular}
    \label{tab:angular_uncertainty}
```

    \caption{Angular uncertainty as a function of distance}
\end{table}

Note that $\sigma_\theta \sim \frac{1}{\mathbf{B}_\perp}$; as the baseline gets larger, the uncertainty decreases.

> \textcolor{red}{add a brief computation for the values from \ref{sec:technical_feasibility_and_heritage} ($\sigma_t = 100 {\ \rm ns}$, $\sigma_{B, ||} \approx 10 {\ \rm m}$)}

Reviewer: ```suggestion

```

	\sigma_\theta = \frac{1}{|\mathbf{B}_\perp|} \sqrt{(c \sigma_t)^2 + \sigma_{B, ||} ^2}.
\end{equation}

> For reference, given $\sigma_t = 100 \ \mathrm{ns}$ and $\sigma_{B, \parallel} \approx 10 \ \mathrm{m}$, we obtain the values in table \ref{tab:angular_uncertainty}.

Reviewer: ```suggestion
For reference, given $\sigma_t = 100 {\ \mathrm{ns}}$ and $\sigma_{B, \parallel} \approx 10 {\ \mathrm{m}}$, we obtain the values in table \ref{tab:angular_uncertainty}.
```

\begin{table}[h]
    \centering
    \begin{tabular}{cccc}
        \hline
        \textbf{Case} & \textbf{Distance (m)} & $\boldsymbol{\sigma_\theta}$ \textbf{(rad)} & $\boldsymbol{\sigma_\theta}$ \textbf{(degrees)} \\
        \hline
        1 & 100  & 0.316  & 18.1 \\
        2 & 250  & 0.126  & 7.22 \\
        3 & 500  & 0.0632 & 3.62 \\
        4 & 1000 & 0.0316 & 1.81 \\
        \hline
    \end{tabular}
    \label{tab:angular_uncertainty}

>     \caption{Angular uncertainty as a function of distance}

Reviewer: ```suggestion
    \caption{Angular uncertainty as a function of satellite baseline distance}
```
